guides/common/modules/proc_salt-configuring-salt.adoc

```diff
@@ -0,0 +1,16 @@
+[id="salt_guide_configuring_salt_{context}"]
+= Configuring Salt
+
+Once you've installed the Salt plugin, you need to connect it to a Salt Master.
+This is required when adding Salt support to an existing {Project} installation, when adding an existing Salt Master to {Project}, or when setting up Salt on {SmartProxy}.
+
+ifdef::orcharhino[]
+[NOTE]
+====
+If you select Salt during step five of the xref:sources/installation_and_maintenance/orcharhino_installation_guide.adoc#oig_main_installation_steps[main orcharhino installation steps], the installer automatically performs those steps on your {Project}.
```

Comment:
Are you sure you want to lock yourself into a step number?

Reply:
Removed `step five of`.